let (foo = func () { let (bar = 'baz') throw {[]} }) try { foo() } catch { func (exc) { "bar" } }
